templates/shoulda_matchers_config_rspec.rb

```diff
@@ -1,3 +1,5 @@
+require 'bundler/setup'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.